multik-api/src/commonTest/kotlin/org/jetbrains/kotlinx/multik/iterable/IterableNDArrayTest.kt

```diff
@@ -15,7 +15,7 @@ import kotlin.test.assertTrue
 class IterableNDArrayTest {
 
     @Test
-    fun `test of function associate`() {
+    fun `test_of_function_associate`() {
```

Comment:
can write without snake_case.

Reply:
You can. But JS does not allow spaces in the test function names. So I can either do something like this or add a @JsName(...) annotation for each